content/getting-started/index.md

```diff
@@ -1,343 +1,37 @@
 ---
-title: Programming the Clawbot
+title: Getting Started with PROS
 ---
 
-#### Objective:
-This tutorial will guide you through basic programming of the VEX Clawbot.
+## Should I use PROS?
+Since you're here and reading our documentation, the answer is most likely yes! However, PROS (and C programming in general) can be difficult for a lot of beginners, so it may not be the ideal solution for everyone.
 
-#### Intended Audience:
-This tutorial is intended for developers with some programming experience, but with little to no experience with the PROS library. If you haven't programmed before, we recommend checking out all the "Introduction and Basic C Features" sections of [this tutorial series](http://www.cprogramming.com/tutorial/c-tutorial.html); you may also benefit from the "Pointers, Arrays and Strings" sections as well (although they aren't as pertintent).
+If you find yourself saying the following things often:
 
-#### Goals:
-At the end of this tutorial you will have:
+* "I don't want to actually learn to code, I just want some code to make my awesome mechanical design move!"
+* "I'm not sure I like coding, but that PROS Editor's dark theme though..."
+* "Reading documentation is for losers, I'm sure I can get my code working with trial and error!"
 
-- Understood the basic project structure of PROS
-- Programmed a basic chassis with "tank" control or "arcade" control
-- Programmed buttons to control the clawbot's lift
-- Programmed a joystick axis to control the clawbot's claw
-- Understood the standard subsystem module methodology
-- Programmed a dead-reckoned autonomous routine
+...then PROS may not be the best choice for programming your robot.
 
+But if you're one of the lovely people who say the following things:
 
-Here's the robot we'll be programming:
-{{< figure src="/images/tuts/clawbot1.jpg" >}}
+* "Writing code may not be easy, but I'm determined to put in the effort required to write great code!"

```

Comment:
Ditto. There's a better way to say this. It feels somewhat like "PROS is inherently hard", when it's not really unless you're brand new to coding.

Reply:
Is there any part of this idea that you would particularly like to keep, or would it be easiest to just strike this one and stick with the other four reasons for using PROS?

Reply:
I think my comment above applies here, too, in that it appears to imply that people who shouldn't/don't use PROS are somehow inferior. Again, we should avoid such implications.
If you can revise it but keep the same general idea (i.e. there may be a somewhat steep learning curve), then it should stay. Otherwise just strike it.

Reply:
I'm thinking the best bet may be to just strike it, we don't want to really emphasize the learning curve from what I can gather from Elliot's comment. There are plenty of other valid reasons for using PROS.

Reply:
Sounds good to me